tests/model_registry/test_model_registry_creation.py

```diff
@@ -1,81 +1,107 @@
-        model_registry_instance_rest_endpoint: str,
-        current_client_token: str,
-        updated_dsc_component_state: DataScienceCluster,
+        model_registry_client: ModelRegistryClient,
+        registered_model: RegisteredModel,
     ):
-        # address and port need to be split in the client instantiation
-        server, port = model_registry_instance_rest_endpoint.split(":")
-        registry = ModelRegistry(
-            server_address=f"{Protocols.HTTPS}://{server}",
-            port=port,
-            author="opendatahub-test",
-            user_token=current_client_token,
-            is_secure=False,
-        )
-        model = registry.register_model(
-            name=MODEL_NAME,
-            uri="https://storage-place.my-company.com",
-            version="2.0.0",
-            description="lorem ipsum",
-            model_format_name=ModelFormat.ONNX,
-            model_format_version="1",
-            storage_key="my-data-connection",
-            storage_path="path/to/model",
-            metadata={
-                "int_key": 1,
-                "bool_key": False,
-                "float_key": 3.14,
-                "str_key": "str_value",
-            },
-        )
-        registered_model = registry.get_registered_model(MODEL_NAME)
+        model = model_registry_client.get_registered_model(MODEL_NAME)
         errors = []
         if not registered_model.id == model.id:
-            errors.append(f"Unexpected id, received {registered_model.id}")
+            errors.append(f"Unexpected id, received {model.id}")
         if not registered_model.name == model.name:
-            errors.append(f"Unexpected name, received {registered_model.name}")
+            errors.append(f"Unexpected name, received {model.name}")
         if not registered_model.description == model.description:
-            errors.append(f"Unexpected description, received {registered_model.description}")
+            errors.append(f"Unexpected description, received {model.description}")
         if not registered_model.owner == model.owner:
-            errors.append(f"Unexpected owner, received {registered_model.owner}")
+            errors.append(f"Unexpected owner, received {model.owner}")
         if not registered_model.state == model.state:
-            errors.append(f"Unexpected state, received {registered_model.state}")
+            errors.append(f"Unexpected state, received {model.state}")
 
         assert not errors, "errors found in model registry response validation:\n{}".format("\n".join(errors))
 
+    def test_model_registry_operator_env(
+        self,
+        admin_client: DynamicClient,
+        model_registry_namespace: Namespace,
+        updated_dsc_component_state_scope_class: DataScienceCluster,
+    ):
+        operator_pod = [
+            pod
+            for pod in Pod.get(dyn_client=admin_client, namespace=py_config["applications_namespace"])
+            if re.match(MR_OPERATOR_NAME, pod.name)
+        ][0]
```

Comment:
should be in a fixture
don't you want to test to see that there's only 1?

Reply:
Fixture: makes sense. We can check for only 1 pod, but is more like a cluster sanity check, that should be part of the entire model registry validation. I plan to add this either as a part of this PR or a future PR.